pkg/kubernetes/kubernetes.go

```diff
@@ -70,16 +72,22 @@ func New(s v1alpha1.Spec) (*Kubernetes, error) {
 type ApplyOpts client.ApplyOpts
 
 // Apply receives a state object generated using `Reconcile()` and may apply it to the target system
-func (k *Kubernetes) Apply(state manifest.List, opts ApplyOpts) error {
+func (k *Kubernetes) Apply(baseDir string, state manifest.List, opts ApplyOpts) error {
```

Comment:
`pkg/kubernetes` should not be concerned with directories and such ... that's the concern of `pkg/tanka`. 

For calculating the environment label ... the environments name is available in `k *Kubernetes`.

Reply:
I'll check that. The label needs to be k8s compatible, so must exclude slashes, hence this label has slashes replaced with dots.

Reply:
I don't see the environment preset in `k *Kubernetes`, but... we could put it there.

Reply:
Alright, it only includes the spec. I’m fine with adding the whole env in there